packages/patternfly-4/react-core/src/components/ClipboardCopy/ClipboardCopy.md

```diff
@@ -0,0 +1,47 @@
+---
+
+title: 'ClipboardCopy'
+cssPrefix: 'pf-c-copyclipboard'
+
+---
+
+## Clipboard Copy
+
+```js
+import React from 'react';
+import { ClipboardCopy } from '@patternfly/react-core';
+class SimpleClipboardCopy extends React.Component {
+  render() {
+    return <ClipboardCopy>This is editable</ClipboardCopy>;
+  }
+}
+```
+
+## Read Only Clipboard Copy
+
+```js
+import React from 'react';
+import { ClipboardCopy } from '@patternfly/react-core';
+class SimpleClipboardCopy extends React.Component {
+  render() {
+    return <ClipboardCopy isReadOnly>This is editable</ClipboardCopy>;
```

Comment:
Should this say "not editable" since it's read only?

Reply:
Dang! You're right!